### ISSUE TITLE
barcodes missing 

### ISSUE THREAD

**Issue by MarieLataretu** [wdecoster/nanocomp#68]
Hi there,

I have a run/sequence summary file with 96 barcodes.

Somehow only the first 50 barcodes are plotted. Also, unclassified is missing.

My command:
```
NanoComp -t 10 -o summary_${summary.baseName} --barcoded --summary ${summary}
```
I'm using version `1.22.0`

I ran NanoComp on the same input some weeks ago (version `1.21`). After checking again, I noticed that barcode 96 is missing (unclassified is there).

For runs with 24 barcodes, `1.22.0` works fine.
The log looks unsuspicious - it's the same as from the 24-barcode run

PS: nice, the sorting seems to work :)

**Comment by wdecoster** [comment]
Aha I have an idea of what's wrong. I'll look into it.

**Comment by wdecoster** [comment]
Should be solved in v1.23.0 - which will soon be available through pip and conda :) Please let me know if you find more weird things :-D

**Comment by MarieLataretu** [comment]
Thanks, works like a charm! 🙂 🚀
